t/parse_version.t

```diff
@@ -110,7 +110,7 @@ our $VERSION = 2.34;
 END
 }
 
-if ( $] > 5.009 && $] < 5.012 ) {
+if ( $] lt '5.012' ) {
```

Comment:
Similarly should remain a numerical comparison.

Reply:
Fixed.
I'll force push it once CI completes successfully.